### PR TITLE
DialogContent component for sizing dialogs

### PR DESCRIPTION
**Pull request by kristofferj**
- DateTime input dialog as now working
- DialogContent component is now a part and can be used to size content
- minor cleanup

### CODE REVIEW

File: packages/@sanity/components/src/dialogs/styles/PopOver.css [hunk 1]
```diff
@@ -234,11 +230,11 @@
   font-weight: 400;
   border-radius: var(--border-radius-base) var(--border-radius-base) 0 0;
 
-  @media (--max-screen-medium) {
+  /* @media (--max-screen-medium) {
```
Comment (bjoerge) [inline packages/@sanity/components/src/dialogs/styles/PopOver.css]: Should these be commented out?

File: packages/@sanity/form-builder/src/inputs/ArrayInput/ItemValue.js [hunk 2]
```diff
@@ -210,6 +211,8 @@ export default class RenderItemValue extends React.Component<Props> {
       )
     }
 
+    const dialogSize = 'medium'
```
Comment (bjoerge) [inline packages/@sanity/form-builder/src/inputs/ArrayInput/ItemValue.js]: This is supposed to be hard coded?

File: packages/@sanity/components/src/dialogs/styles/PopOver.css (hunk 1 above)
Comment (kristofferj) [reply packages/@sanity/components/src/dialogs/styles/PopOver.css]: No. Pushing a fix now.

File: packages/@sanity/form-builder/src/inputs/ArrayInput/ItemValue.js (hunk 2 above)
Comment (kristofferj) [reply packages/@sanity/form-builder/src/inputs/ArrayInput/ItemValue.js]: Yes. Until we have a resolver for size. This sets the size for both popover and defaultdialog

**Comment by kristofferj** [comment]
I will test this a little bit before i merge it.

**Comment by kristofferj** [comment]
Rebased against next

**Pull request merged**